Gemfile.lock

```diff
@@ -59,27 +72,39 @@ GEM
       forwardable-extended (~> 2.6)
     psych (4.0.3)
       stringio
-    public_suffix (4.0.6)
+    psych (4.0.3-java)
+      jar-dependencies (>= 0.1.7)
+    public_suffix (5.0.4)
     rake (13.0.6)
-    rb-fsevent (0.11.0)
+    rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
     rdoc (6.4.0)
       psych (>= 4.0.0)
-    rexml (3.2.5)
-    rouge (3.27.0)
+    rexml (3.2.6)
+    rouge (4.2.0)
     safe_yaml (1.0.5)
-    sassc (2.4.0)
-      ffi (~> 1.9)
+    sass-embedded (1.69.5)
+      google-protobuf (~> 3.23)
+      rake (>= 13.0.0)
+    sass-embedded (1.69.5-arm-linux-gnueabihf)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.69.5-arm64-darwin)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.69.5-x86_64-linux-gnu)
+      google-protobuf (~> 3.23)
     stringio (3.0.1)
-    terminal-table (2.0.0)
-      unicode-display_width (~> 1.1, >= 1.1.1)
+    terminal-table (3.0.2)
+      unicode-display_width (>= 1.1.1, < 3)
     tomlrb (2.0.1)
-    unicode-display_width (1.8.0)
+    unicode-display_width (2.5.0)
     webrick (1.7.0)
 
 PLATFORMS
-  arm64-darwin-21
+  arm64-darwin
+  arm64-linux
+  java
```

Comment:
:question: :thinking:

Reply:
copied from another one of our lockfiles